src/role_carry.js

```diff
@@ -68,13 +68,13 @@ roles.carry.preMove = function(creep, directions) {
   if (!creep.memory.routing.reverse) {
     reverse = creep.checkForTransfer(directions.forwardDirection);
   }
-
+  // less twiddling around
   let carryPercentage = 0.1;
   if (creep.room.name == creep.memory.routing.targetRoom) {
-    carryPercentage = 0.8;
+    carryPercentage = 0.5;
```

Comment:
Maybe extract these values to `config.js`. Easier to tweak around and to overwrite.